### PR TITLE
completed Assignment

### PR DESCRIPTION
**Pull request by emthoendel**
I'm creating this pull request so that I can review your code line by line. In future assignments, I will expect you to submit this or dock you points.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Assignment_01 </title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (emthoendel) [inline index.html:5]: This references a CDN hosted version of jQuery - which is fine! You don't need the jQuery file though then - but maybe you were using it in development?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <!---- <script src="jquery-3.4.1.min.js"></script>-->
+        <script src="myscript.js"> </script>
```
Comment (emthoendel) [inline index.html:7]: custom js file! awesome!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    </head>
+    <body>
+
+        <!--     
+            Deepa Natarajan
+            assignment01.htm
+            INFO_2340_WW
+            Ms.Thoendel
+            12/08/2019
+        -->
```
Comment (emthoendel) [inline index.html:18]: Super happy to see this comment block!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+               
+        </script>
+
+        <h1>Greetings</h1>
+        <p> Hello my name is <span id="firstname"></span><span id="lastname"></span></p>
+        <p> I am a student at <span class="mcc">Metropolitan Community College</span> in Omaha, Nebraska !</p>
+        <p>My Hobbies include: <br>
+            <ul>
+                <li id="hobby">Walking</li>
```
Comment (emthoendel) [inline index.html:27]: Just like in figure 1! Perfect!

File: myscript.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
```
Comment (emthoendel) [inline myscript.js:1]: nice anonymous function to get called a document ready

File: myscript.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+    $("#firstname").html("Deepa");
+    $("#lastname").html("Natarajan !");
+    $(".mcc").css('color','blue');
+    $(".mcc").css('font-style','italic');
+    $("#hobby").append("<li>Painting</li>");
+    $("#hobby").append("<li>Playing with my Kid</li>");
```
Comment (emthoendel) [inline myscript.js:8]: excellent job of using jQuery to target an element by ID and then append/alter content! You've also successfully added style for the mcc class! Well done!

**Review by emthoendel — COMMENTED** [review]
Great code! It looks like your biggest struggle was getting the code submitted properly with GitHub as a pull request. Let me know if you have questions. You'll want to make sure to have it down for this next assignment.

File: index.html (hunk 3 above)
Comment (deepa-Natarajan) [reply index.html:18]: I am super happy too, to see all your comments. It is very lively, I love this class structure very much.Thank you!!

**Comment by emthoendel** [comment]
I’m so glad you like the comments! I do, too – its why I try to use GitHub for any of my programming classes.

I hope this week goes just as well for you!

Lisa

**Pull request merged**